widgets/accordion/js/accordion.js

```diff
@@ -33,6 +33,10 @@ jQuery( function ( $ ) {
 					);
 					$panel.addClass( 'sow-accordion-panel-open' );
 					openPanels.push( panel );
+					parentPanel = $( panel ).parents( '.sow-accordion-panel' );
+					if ( parentPanel.length && openPanels.indexOf( parentPanel.get(0) ) == -1 ) {
+						openPanels.push( parentPanel.get(0) );
```

Comment:
I don't think we should be mixing parent panels in with child panels here. Seems like this could cause the max open panels issue I'm seeing with parent accordion. How are you supposed to tell the parent and child panels apart? What if they have the same names? Just thinking that could also cause confusion in the URL, if anchors are the same for parent and child.